vdsm_hooks/README.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Why not add the header to the actual README?

.license files is neded for binary files or files that cannot have comments. Plain text files can have comments.

Reply:
Right, but then the license header would be visible in the repository as plain text, as it is not markdown, or asciidoc: https://github.com/oVirt/vdsm/tree/master/vdsm_hooks

Reply:
Same as hook, visible headers is fine and can be improved later if needed.